src/Microsoft.Azure.WebJobs.Host/Config/IWebJobsExtensionOptionsConfiguration.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.Options;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Host.Config
+{
+    public interface IWebJobsExtensionOptionsConfiguration<TOptions> : IConfigureOptions<TOptions> where TOptions : class
```

Comment:
This interface shouldn't be generic - remove the <TOptions> from it

Reply:
Hi @mathewc  
I think we need it. The following code is usage of the Azure Functions Host side. If we have multiple extensions registered, how can we identify it? If you know the better way, please let me know. 

https://github.com/Azure/azure-functions-host/blob/178dc626799e747be0b9575ebe2d380b868eca35/src/WebJobs.Script.WebHost/HostJsonConfigProvider.cs#L77-L81

Reply:
Done

Reply:
Done